src/Dapr.SecretStore.ConfigurationExtension/DaprSecretDescriptor.cs

```diff
@@ -0,0 +1,41 @@
+using System;
```

Comment:
Add copyright headers, please see the files in current code base to get the header.

Reply:
Solved in e2b1b2e